app/frontend/src/features/auth/signup/Signup.tsx

```diff
@@ -1,33 +1,133 @@
-import React from "react";
-import { Redirect, Route, Switch } from "react-router-dom";
+import { Box, makeStyles, Typography } from "@material-ui/core";
+import React, { useEffect } from "react";
+import { Link, Redirect, Route, Switch } from "react-router-dom";
 
 import Alert from "../../../components/Alert";
-import PageTitle from "../../../components/PageTitle";
-import { signupRoute } from "../../../routes";
+import AuthHeader from "../../../components/AuthHeader";
+import { loginRoute, signupRoute } from "../../../routes";
 import { useAuthContext } from "../AuthProvider";
+import AuthBg from "../resources/auth-bg.png";
 import CompleteSignupForm from "./CompleteSignupForm";
 import EmailForm from "./EmailForm";
 
+const useStyles = makeStyles((theme) => ({
+  signupPage: {
+    boxSizing: "border-box",
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "center",
+    height: "100vh",
```

Comment:
Because of this, the vertical padding has no effect at the bottom of long pages like the CompleteSignup page

Reply:
Did you encounter a problem that form didn't fit in the screen?